deployment/deploymentConfigurations.json

```diff
@@ -72,6 +72,14 @@
           "dmelotik": "dmelotik/aave-v2-rwa",
           "deploy-on-merge": true
         }
+      },
+      "geist": {
+        "fantom": {
+          "template": "geist.template.yaml",
+          "messari": "messari/geist-fantom",
```

Comment:
our subgraph is call `geist-finance-fantom`:
https://thegraph.com/hosted-service/subgraph/messari/geist-finance-fantom